scripts/superdesk-packaging/packaging.less

```diff
@@ -214,6 +214,9 @@
                 padding: 40px 0;
             }
         }
+        .preview-container {
+        	padding: 20px 20px;
```

Comment:
this file have mixed tab size (sometimes it's 4 spaces, sometimes 4 tabs). Can you fix it? Even in this line it's 8 spaces and 1 tab.